### DISCUSSION TITLE
Applied as Teacher but accepted as Student

### DISCUSSION THREAD

**Discussion by kschulzke** [community/community#82725] [GitHub Education]
### Hi everyone,

I applied to Github Global Campus as a Teacher but was accepted as a Student. Submitted a support ticket but have received no response.

**Comment by mahfuz0001** [comment]
You might have to have patience until they response to your ticket

**Reply by kschulzke** [reply]
"Patience" is probably not the right word. Still waiting for a response.

**Comment by hejoefs** [comment]
Hey Kurt,

I'm sorry about the mix-up with your application to GitHub Global Campus. Sometimes, these administrative errors can be frustrating. While waiting for a response to your support ticket, consider reaching out through multiple channels if possible.

If you are still waiting to hear back after a reasonable amount of time, consider following up again or looking for alternative contact methods or support avenues within the GitHub Global Campus program. These situations can be inconvenient, but persistence and clear communication often help resolve them.

**Reply by kschulzke** [reply]
Umm. Dude. I have gone through multiple channels. This is one of them. And it should be very clear what happened here.
